routes/chat.js

```diff
@@ -143,4 +144,6 @@ router.post(
   leaveGroupMember
 );
 
+router.get('/message/:messageID', auth.isAuth, getMessageDetail);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new route `/message/:messageID` is added correctly. However, it would be better to validate the `messageID` parameter using Joi validation middleware before passing it to the `getMessageDetail` function. This will ensure that only valid message IDs are passed to the function, improving security and reducing unnecessary API calls.
```diff
- router.get('/message/:messageID', auth.isAuth, getMessageDetail);
+ router.get('/message/:messageID', auth.isAuth, validate(ChatValidation.validateMessageId), getMessageDetail);
```
Please make sure to add a `validateMessageId` schema in `ChatValidation`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->